ForumCrawler/ForumCrawler.csproj

```diff
@@ -374,9 +374,6 @@
     <Compile Include="Migrations\202005091205228_RemoveExemptAndShowScore.Designer.cs">
       <DependentUpon>202005091205228_RemoveExemptAndShowScore.cs</DependentUpon>
     </Compile>
-    <Compile Include="Migrations\202005102158383_InertiaRevamp.cs" />
-    <Compile Include="Migrations\202005102158383_InertiaRevamp.Designer.cs">
-      <DependentUpon>202005102158383_InertiaRevamp.cs</DependentUpon>
```

Comment:
Migrations are purely for tracking the history of the database. What you're doing is the equivalent of removing a specific commit in the timeline of a git repository - if your aim is to remove inertia from the database, create a new migration which drops the entry.